x-pack/plugins/session_view/server/routes/io_events_route.ts

```diff
@@ -88,43 +88,47 @@ export const searchProcessWithIOEvents = async (
       ]
     : [];
 
-  const search = await client.search({
-    index: [PROCESS_EVENTS_INDEX],
-    body: {
-      query: {
-        bool: {
-          must: [
-            { term: { [EVENT_ACTION]: 'text_output' } },
-            { term: { [ENTRY_SESSION_ENTITY_ID_PROPERTY]: sessionEntityId } },
-            ...rangeFilter,
-          ],
+  try {
+    const search = await client.search({
+      index: [PROCESS_EVENTS_INDEX],
+      body: {
+        query: {
+          bool: {
```

Comment:
Maybe also add a filter to filter out bad-data on the index that could prevent the `agg.buckets` from being created in such circumstances

```js
"bool": {
    "filter": [
        {
            "exists": { "field": PROCESS_ENTITY_ID_PROPERTY }
        }
    ],
    "must_not": [
        {
            "match_phrase": { PROCESS_ENTITY_ID_PROPERTY: "" }
        }
    ]
}
```